logic app templates/readme.md

```diff
@@ -0,0 +1,23 @@
+# DFP Event Tracing Event Hubs Logic App Templates
+## Overview
+These are a set of Logic App templates that work specifically with DFP events received inside Event Hubs for getting started quickly with working with DFP data. 
+
+## Reference Links
+[DFP Event Tracing Documentation](https://docs.microsoft.com/en-us/dynamics365/fraud-protection/event-tracing)
+
+## Setup
+
+**Audit Events**:
+
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fmicrosoft%2FDynamics-365-Fraud-Protection-Samples%2Fkhavnguyen%2Fevent-tracing-logic-app-templates%2Flogic%2520app%2520templates%2Faudit-events-template.json)
```

Comment:
I think you can derive this ahead of time.  It's going to be:

https://raw.githubusercontent.com/microsoft/Dynamics-365-Fraud-Protection-Samples/master/logic%20app%20templates/audit-events-template.json

once merged.  Not sure if you need to keep the URL escaped, but that will be the path to it and a similar one for the latency.  Do you mind optimistically changing these and we can always update if that's not right for some reason?

Reply:
Will do this for both links. Hope it works.